db/migrate/20220720214646_add_delete_cascade_from_orgs_down.rb

```diff
@@ -0,0 +1,9 @@
+class AddDeleteCascadeFromOrgsDown < ActiveRecord::Migration[7.0]
+  def up
+    add_foreign_key :schemes, :organisations, column: "owning_organisation_id", on_delete: :cascade
```

Comment:
Does this work if a foreign key already exists or do you need to remove the existing one first?

(i.e. if you do `rake db:drop; rake db:create; rake db:migrate` is it successful?)

Reply:
It does now!